src/objects/Mesh.js

```diff
@@ -135,17 +135,7 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		function checkIntersection( object, material, raycaster, ray, pA, pB, pC, point ) {
 
-			var intersect;
-
-			if ( material.side === BackSide ) {
-
-				intersect = ray.intersectTriangle( pC, pB, pA, true, point );
-
-			} else {
-
-				intersect = ray.intersectTriangle( pA, pB, pC, material.side !== DoubleSide, point );
-
-			}
+			var intersect = ray.intersectTriangle( pA, pB, pC, material.side !== DoubleSide, point );
```

Comment:
I'm afraid this change is not correct. Notice the difference in the order of parameters in the first and second call of `intersectTriangle()`:
```js
pC, pB, pA
```
and
```
pA, pB, pC
```